src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -627,28 +627,28 @@
 examples:
   - name: Create a role with read-only access to storage and network resources, and the ability to start or restart VMs.
     text: |
-        az role definition create --role-definition '{ \\
-            "Name": "Contoso On-call", \\
-            "Description": "Perform VM actions and read storage and network information.", \\
-            "Actions": [ \\
-                "Microsoft.Compute/*/read", \\
-                "Microsoft.Compute/virtualMachines/start/action", \\
-                "Microsoft.Compute/virtualMachines/restart/action", \\
-                "Microsoft.Network/*/read", \\
-                "Microsoft.Storage/*/read", \\
-                "Microsoft.Authorization/*/read", \\
-                "Microsoft.Resources/subscriptions/resourceGroups/read", \\
-                "Microsoft.Resources/subscriptions/resourceGroups/resources/read", \\
-                "Microsoft.Insights/alertRules/*", \\
-                "Microsoft.Support/*" \\
-            ], \\
-            "DataActions": [ \\
-                "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/*" \\
-            ], \\
-            "NotDataActions": [ \\
-                "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write" \\
-            ], \\
-            "AssignableScopes": ["/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"] \\
+        az role definition create --role-definition '{ 
```

Comment:
The CI will fail since there is a trailing space:

```suggestion
        az role definition create --role-definition '{
```

Reply:
Got it, fixed already.